.github/workflows/api-compatibility.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: ~1.19.6
```

Comment:
shouldn't this be `>=` instead of `~`? Looking at https://github.com/actions/setup-go#v3